integration-test/990-add-art-galleries.py

```diff
@@ -0,0 +1,9 @@
+# http://www.openstreetmap.org/node/2026996113
+assert_has_feature(
+    17, 20971, 50656, 'pois',
```

Comment:
Our max_zoom is 16, so you'd test the zoom 16 tile and see if the min_zoom on the feature is 17.